scripts/ci/github_actions/driver/setup_ci_dependencies.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -xe
+set +o pipefail
+
+install_ci_dependencies (){
+  python -m pip install --upgrade pip
+  echo docker-hub==2.2.0 > dev-requirements.txt
+  pip install -r dev-requirements.txt
+cat >>/home/runner/.bash_profile <<'EOL'
+yq() {
+  docker run --rm -i -v "${PWD}":/workdir mikefarah/yq "$@"
+}
+EOL
+}
+
+get_driver_version (){
+  source /home/runner/.bash_profile
```

Comment:
shouldn't this be done earlier? e.g. at `install_ci_dependencies`

Reply:
done